src/types/ScenarioAttributes.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
File be gone

Reply:
Haha yes, sorry I guess I didn't pull the most recent version